Makefile

```diff
@@ -23,3 +23,27 @@ torchtext: lib
 	cd build/torchtext
 	git submodule update --init --recursive
 	python setup.py clean install
+
+cu126:
+	#nvcc -V  # See original CUDA versions
+	sudo apt-get update
+	#sudo dpkg -l | grep -E "nvidia|cuda"  # See related packages
+	sudo apt-get --allow-change-held-packages --purge remove "nvidia*" "libnvidia-*" "*cublas*" "cuda*" "nsight*"
+	sudo rm -rf /usr/bin/nvidia*
+	sudo rm -rf /usr/lib64-nvidia/*  # This path is Colab's weird choice and the folder is locked by other processes
+	sudo rm -rf /opt/nvidia
+	sudo rm -rf /opt/bin/nvidia-*
+	sudo apt-get autoremove -y
+	sudo apt-get autoclean
+	wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64/cuda-keyring_1.1-1_all.deb
+	sudo dpkg -i cuda-keyring_1.1-1_all.deb
+	sudo apt-get update
+	sudo apt-get -y install cuda-12-6
+	sudo apt-get autoremove -y
+	sudo apt-get autoclean
+	#nvcc -V  # See updated CUDA versions
+	sudo update-alternatives --display cuda
+
+pt260: cu126
+	pip uninstall -y torch torchaudio torchvision
+	pip install "torch>=2.6.0,<2.7" torchaudio torchvision --index-url https://download.pytorch.org/whl/cu126
```

Comment:
In the near future this has to be more cooperative with `constraints.txt` and `requirements.txt`.